SqlDatabases/Edition.cs

```diff
@@ -0,0 +1,19 @@
+namespace Linq2Azure.SqlDatabases
+{
+    public class Edition
+    {
+
+        private Edition(string value)
+        {
+            Value = value;
+        }
+
+        public string Value { get; private set; }
+
+        public static Edition Web = new Edition("Web");
```

Comment:
Should we be flagging these as legacy somehow, or is that not "our job" to do that (i.e. let the api output indicate what is getting deprecated etc)